swan-lake/development-tutorials/observability/supported-observability-tools-and-platforms/datadog.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Observe Ballerina programs with DataDog
+description: See how Ballerina supports observability by exposing itself via metrics and tracing to DataDog.
+keywords: ballerina, observability, metrics, tracing, datadog
+permalink: /learn/supported-observability-tools-and-platforms/datadog/
+active: datadog
+intro: The sections below include information about how to observe Ballerina programs with DataDog.
```

Comment:
Since this line also appears as the first line of the page, the content seems duplicated.

Reply:
Addressed review suggestions